### PR TITLE
MCR-2535 Mitigate CVE-2021-44228

### PR DESCRIPTION
**Pull request by toKrause**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2535).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,8 +10,8 @@
   <version>2021.06.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>MyCoRe</name>
-  <description xml:space="preserve">
-MyCoRe is a system for the development of institutional content repositories, publication servers, image collections, archives, or similar repositories.
+  <description xml:space="preserve">
```
Comment (rsteph-de) [inline pom.xml]: This is probably a linebreak issue - Can you remove this change from PR

Comment (toKrause) [reply pom.xml]: I have no idea whats happening there. Whenever I save the file (IDE or normal editor), git detects a change, ... `git add -p` to the rescue

Comment (toKrause) [reply pom.xml]: You're right, for whatever reason the is a Windows style linebreak (`0xD 0xA`) after `"preserve">`

**Review by rsteph-de — APPROVED** [review]
Looks good for me!
Please merge ASAP!

**Pull request merged**